.github/workflows/static-analysis.yml

```diff
@@ -49,6 +49,8 @@ jobs:
 
       - name: "Install dependencies with Composer"
         uses: "ramsey/composer-install@v3"
+        with:
+          composer-options: "--ignore-platform-req=php+"
```

Comment:
This is necessary because of Psalm, right?

Reply:
Yes. I know, we need to kick it.